frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/CodeLists/CodeLists.tsx

```diff
@@ -43,3 +44,10 @@ function CodeList({ codeList, onUpdateCodeList }: CodeListProps) {
     </Accordion>
   );
 }
+
+export const updateCodeListWithMetadata = (
+  codeListTitle: string,
+  updatedCodeList: StudioComponentsCodeList,
+): CodeListWithMetadata => {
+  return { title: codeListTitle, codeList: updatedCodeList };
+};
```

Comment:
Dette fungerer så lenge objektet bare har disse to egenskapene, men det er ikke skalerbart. Hvis vi legger til nye egenskaper på `CodeListWithMetadata`, må vi også tilpasse funksjonen.

Hva tenker du om å ta inn hele det gamle objektet i en parameter og sørge for at det som kommer ut er en kopi hor kun kodelisten er oppdatert?
```suggestion
export const updateCodeListWithMetadata = (
  currentCodeListWithMetadata: CodeListWithMetadata,
  updatedCodeList: StudioComponentsCodeList,
): CodeListWithMetadata => ({ ...currentCodeListWithMetadata, codeList: updatedCodeList });
```